packages/@sanity/codegen/src/readConfig.ts

```diff
@@ -0,0 +1,28 @@
+import {readFile} from 'fs/promises'
```

Comment:
I realize we should have eslint enforce this rather than randomly calling it out in code reviews, so feel free to ignore, but would be great to have consistent `node:`-prefixed imports.